helix-core/src/test/java/org/apache/helix/ZkTestHelper.java

```diff
@@ -202,7 +202,7 @@ public void process(WatchedEvent event) {
 
     String newSessionId = Long.toHexString(curZookeeper.getSessionId());
     LOG.info("After session expiry. sessionId: " + newSessionId + ", zk: " + curZookeeper);
-    Assert.assertNotSame(newSessionId, oldSessionId, "Fail to expire current session, zk: "
+    Assert.assertFalse(newSessionId.equals(oldSessionId), "Fail to expire current session, zk: "
```

Comment:
What is difference here?

Reply:
`assertNotSame` checks two objects don't refer to the same object. But here we actually would like to check if these two sessions are equaled or not. If we use `assertNotSame`, this would always pass because these two sessionId strings are different objects.